code/modules/clothing/spacesuits/rig/modules/utility.dm

```diff
@@ -121,22 +122,22 @@
 	selectable = 0
 	toggleable = 0
 	disruptive = 0
-	use_power_cost = 500
+	use_power_cost = 5000
 
 	engage_string = "Inject"
 
 	interface_name = "integrated chemical dispenser"
 	interface_desc = "Dispenses loaded chemicals directly into the wearer's bloodstream."
 
 	charges = list(
-		list("tricordrazine", "tricordrazine", /datum/reagent/tricordrazine,     80),
-		list("dramadol",      "tramadol",      /datum/reagent/tramadol,          80),
-		list("dexalin plus",  "dexalin plus",  /datum/reagent/dexalinp,          80),
-		list("antibiotics",   "antibiotics",   /datum/reagent/spaceacillin,      80),
-		list("antitoxins",    "antitoxins",    /datum/reagent/dylovene,          80),
-		list("glucose",       "glucose",       /datum/reagent/nutriment/glucose, 80),
-		list("hyronalin",     "hyronalin",     /datum/reagent/hyronalin,         80),
-		list("radium",        "radium",        /datum/reagent/radium,            80)
+		list("inaprovaline",  "inaprovaline",  /datum/reagent/inaprovaline,      80),
+		list("dylovene",      "dylovene",      /datum/reagent/dylovene,          80),
```

Comment:
80 inap and dylo means 160 tric, not 80. change it to 40u each

Reply:
It isn't just for tricord. I can make dylo 40 still but inap should stay same.